pipeline.py

```diff
@@ -51,19 +55,27 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path_raw = Path(self.path_to_raw_txt_data)
+
+        files = [file for file in path_raw.glob('*_raw.txt')]
```

Comment:
are you sure this comprehension is needed as soon as you take all elements from iterator? any other (shorter and better) way of obtaining a list here?